app/controllers/salesforce/persons_controller.rb

```diff
@@ -6,7 +6,12 @@ class Salesforce::PersonsController < Admin::AdminController
 
   def create
     begin
-      Salesforce::Person.create!(params.require(:type), user)
+      type = params.require(:type)
```

Comment:
if type is not lead or contact it will return success while doing nothing?

Reply:
Now, it will return an `ArgumentError`